job-templates/job.yaml

```diff
@@ -59,15 +59,15 @@ spec:
             valueFrom: {secretKeyRef: {name: saucelabs-account, key: username}}
           - name: SAUCELABS_API_KEY
             valueFrom: {secretKeyRef: {name: saucelabs-account, key: api-key}}
+          - name: CICD_SCALR_TOKEN
+            valueFrom: {secretKeyRef: {name: scalr-cicd-token, key: token}}
```

Comment:
Didn't see any references to CICD_SCALR_TOKEN and scalr-cicd-token. How are they used?

Reply:
https://github.com/ProjectSigma/sigma-monorepo/pull/6318